src/tabs/default.scss

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import './tabs';
+
+@mixin iui-tabs-default {
+  .iui-tab {
+    height: $iui-component-height;
+    padding: 0 $iui-m;
+    box-sizing: border-box;
+    @include themed {
+      background-color: t(iui-color-background-2);
+      border: 1px solid t(iui-color-background-5);
+    }
+
+    &:hover {
+      @include themed {
+        background-color: t(iui-color-background-1);
+      }
+    }
+
+    &.iui-active {
+      border-bottom-color: transparent;
+      @include themed {
+        background-color: t(iui-color-background-1);
+      }
+    }
+
+    &[disabled] {
+      @include themed {
+        background-color: t(iui-color-background-3);
+      }
+    }
+  }
+
+  li:not(:first-child) .iui-tab {
+    border-left: none;
+  }
+
+  &.iui-green .iui-tab {
+    @include themed {
+      box-shadow: inset inherit inherit inherit inherit t(iui-color-foreground-positive);
```

Comment:
Is it ok that there is so many `box-shadow` rules that overrides each other?
![image](https://user-images.githubusercontent.com/36186912/122521199-3498f300-d01d-11eb-97e4-e5090d1ccdde.png)

Reply:
Removed. The default tabs don't the "not-animated" class (because there is no other variant, there is no stripe), so I've simplified it by setting the box-shadow and transition in default.